src/components/Share.jsx

```diff
@@ -40,12 +41,12 @@ const Share = ({
                     ></div>
                     <div className='content'
                         style={{display: ((!loading && !urlGenerated) ? 'flex' : 'none' )}}>
-                        <div
+                        {shortUrlFailed ? <p>Failed to generate link</p> : <div
```

Comment:
@shayanbinary `t.translate("Failed to generate link")`

Reply:
@aminroosta ooops, such a shameful mistake . Sorry about that

Reply:
No worries, the entire point of code reviews is to catch these simple errors.